src/app/containers/Main.js

```diff
@@ -206,7 +206,7 @@ const Main = ({
       ipcRenderer.removeListener('state-fetched', stateFetchedListener)
     }
     ipcRenderer.on('state-fetched', stateFetchedListener)
-    ipcRenderer.send('pls-fetch-state', win.id, isInProMode)
+    ipcRenderer.send('pls-fetch-last-file', win.id, isInProMode)
```

Comment:
This is what's preventing the file from opening.  We need the reply to `pls-fetch-state` to open a file :)

Reply:
Oh wait, I see what you did later.  Reading further.

Reply:
When you send on the renderer, it'll be heard on the main process.  It looks like you setup the listener for this ipc call in the renderer?